app/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="spinnerItems">
```

Comment:
Just curious, what is this array used for?

Reply:
It was used to populate the spinner with project categories back when that was being used instead of a TabLayout. No need for it now so just deleted it. Thanks for catching that!